src/components/Sidebar/RightSidebar/RightSidebar.module.css

```diff
@@ -18,5 +18,5 @@
 
 .withLanguageNavBar {
   /* NOTE: when we have a language bar, the space on top should also be doubled. */
-  top: calc(var(--top-nav-height) * 2);
+  top: 11rem;
```

Comment:
Are you sure we don't want to use global variable here? We're alos using some calculations in `Sidebar.module.css` I can't remember the exact reasons, but make sure it's in sync. Because if you're taking out the `var(--top-nav-height)` variable here and hardcoding the value, it might affect styles in the `Sidebar` component. 

Again, not sure how, but just FYI 😅

Reply:
@snikidev  you are using, to make it sync I use  the variable back and just + the value of the top that was added. thanks